megamek/i18n/megamek/common/options/messages.properties

```diff
@@ -1,6 +1,8 @@
 GameOptionsInfo.group.basic.displayableName=Basic Options
 GameOptionsInfo.option.friendly_fire.displayableName=Friendly fire
 GameOptionsInfo.option.friendly_fire.description=If checked,\nthe game will allow attacks against friendly units,\ncontrary to the rules.\nUnchecked by default.
+GameOptionsInfo.option.turn_timer.displayableName=Turn timer
+GameOptionsInfo.option.turn_timer.description=If value is greater 0,\n the game will end a players current action turn after the time is up.\n deactivated by 0 value by default.
```

Comment:
The description should say what the number means. From reading the rest of the code and testing I know it's the number of seconds, but there's nothing that tells this to the user.

Reply:
Yes, it should better be in minutes and say so in the description. Will change it that way.

As for the Style-Guide, you are right too ...